topics/data-science/tutorials/ro-crate-intro/tutorial.md

```diff
@@ -0,0 +1,642 @@
+---
+layout: tutorial_hands_on
+
+title: RO-Crate - Introduction
+level: Introductory
+requirements:
+- type: none
+  title: "Files and folder organization"
+- type: none
+  title: "Familiarity with JSON file format and editor/IDE"
+- type: none
+  title: "An overview of the RO-Crate concept and its implementations"
+  url: "https://gallantries.github.io/video-library/videos/ro-crates/intro/slides/"
+- type: "internal"
+  topic_name: data-science
+  tutorials:
+      - cli-basics
+
+license: Apache license, version 2.0 <http://www.apache.org/licenses/LICENSE-2.0>
+
+copyright: © Copyright 2021-2023 University of Technology Sydney, The University of Manchester UK and RO-Crate contributors
+
+follow_up_training: []
+questions:
+- How do I package data in a FAIR way?
+- How can I list the authors of individual files?
+- Can I use multiple licenses in the same data package?
+- How can I visualize JSON-LD metadata?
+
+objectives:
+- Construct an RO-Crate by hand using JSON
+- Describe each part of the Research Object
+- Learn basic JSON-LD to create FAIR metadata
+- Connect different parts of the Research Object using identifiers
+
+time_estimation:  30M
+key_points:
+- RO-Crate provides a structure to make FAIR data packages
+- schema.org in JSON-LD provides a controlled vocabulary for FAIR metadata
+- Each entity of the crate is described separately
+- Cross-references between entities create a graph
+- The RO-Crate specification recommends which types and keys to use
+
+subtopic: fair
+contributors:
+  - stain
+  - ptsefton
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.